package.json

```diff
@@ -680,7 +680,7 @@
 		"fs-extra": "^8.1.0",
 		"jschardet": "2.1.1",
 		"os-locale": "5.0.0",
-		"vsce": "1.57.0",
+		"vsce": "1.88.0",
```

Comment:
this is expected changes? I see this commit is to resovle the underscore package.

Reply:
Yes, as explained in the abstraction, unlike a manual change made on my keyboard, it's resolved by npm.